lighthouse-core/test/lib/manifest-parser-test.js

```diff
@@ -455,4 +455,99 @@ describe('Manifest Parser', function() {
       assert.equal(url0, undefined);
     });
   });
+
+  describe.only('background_color, theme_color', () => {
```

Comment:
remove `.only`

Reply:
d'oh, they should have a test test for that :)